pyk/src/pyk/kastManip.py

```diff
@@ -117,14 +117,14 @@ def simplify_bool(k):
         return None
```

Comment:
Please add a type signature for `simplify_bool`

Reply:
I will get this in follow-on PR.